test/jdk/sun/security/pkcs11/Provider/MultipleLogins.java

```diff
@@ -30,35 +41,38 @@
 import javax.security.auth.callback.PasswordCallback;
 import javax.security.auth.callback.UnsupportedCallbackException;
 import javax.security.auth.login.LoginException;
+
 import java.io.IOException;
 import java.lang.ref.WeakReference;
 import java.security.*;
 
 import jdk.test.lib.util.ForceGC;
 import jtreg.SkippedException;
 
-public class MultipleLogins {
-    private static final String KS_TYPE = "PKCS11";
+public class MultipleLogins extends PKCS11Test {
```

Comment:
Is there any special reason to keep the class standalone and not a subclass of `PKCS11Test`?